crates/tabby-scheduler/src/path.rs

```diff
@@ -0,0 +1,7 @@
+use std::path::PathBuf;
+
+use tabby_common::path::tabby_root;
+
+pub fn incremental_repository_store() -> PathBuf {
+    tabby_root().join("incremental-repositorries.kv")
+}
```

Comment:
```suggestion
pub fn dataset_store() -> PathBuf {
    tabby_root().join("incremental-repositorries.kv")
}
```

this should be in common/path, with `dataset/repository.kv`